code/game/turfs/simulated/floor/plating.dm

```diff
@@ -29,6 +29,7 @@
 		icon_state = icon_plating //Because asteroids are 'platings' too.
 
 /turf/open/floor/plating/attackby(obj/item/C, mob/user, params)
+	. = SendSignal(COMSIG_PARENT_ATTACKBY, C, user, params)
 	if(..())
```

Comment:
Why isn't ..() calling the base proc?

Reply:
because floor has unique stuff that plating doesn't want to do I guess.

Reply:
The component system is meant for every signal to have one point of origin, so I have to request that you refactor this so that calling ..() works

Reply:
this is going to cause so many bugs REEEEEEE

Reply:
This was bait all along to make me handle all of turfs. I see your trick!